release.nix

```diff
@@ -6,7 +6,7 @@ let
 
   pkgs = import <nixpkgs> {};
 
-  systems = [ "x86_64-linux" "i686-linux" "x86_64-darwin" /* "x86_64-freebsd" "i686-freebsd" */ ];
+  systems = [ "x86_64-linux" "i686-linux" "x86_64-darwin" "i686-cygwin" "x86_64-cygwin" /* "x86_64-freebsd" "i686-freebsd" */ ];
```

Comment:
@edolstra i've looked into release.nix again. ``systems`` only lists platforms that nix can be built on. later in the release.nix there is a release attribute  (line 231) and all platforms that hydra will build are listed there. please correct me if i am wrong.